### ISSUE TITLE
Missing Hover Effects on Buttons in User Profile Dashboard

### ISSUE THREAD

**Issue by deepak05kr18** [CircuitVerse/CircuitVerse#4538]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

When accessing the CircuitVerse website and navigating to the user profile dashboard, it has been observed that several buttons lack the expected hover effect. This issue affects the overall user experience, as the absence of hover effects can lead to confusion and may not provide users with the visual feedback they need for interactive elements.

### Steps To Reproduce

1.  Log in to the CircuitVerse website.
2.  Navigate to your user profile dashboard.
3.  Observe the buttons within the dashboard interface.

### Expected Behavior

All buttons within the dashboard should exhibit a clear and consistent hover effect, providing users with a visual cue that the button is interactive.

### Screenshots

I have provided a screen recording illustrating the issue along with its proposed solution.

https://github.com/CircuitVerse/CircuitVerse/assets/122279122/d2c8d83d-45a2-49dc-83c0-293e1ba49556

### Device Information [optional]

```markdown
- OS: Windows
- Browser: Google Chrome
- version: 120.0.6099.199
```

### Are you working on this issue?

Yes

**Comment by pankaj-2503** [comment]
Can you assign this to me i want to work on this

**Comment by pankaj-2503** [comment]
I have added the hover effects on user profile dashboard please review the PR.

**Comment by pankaj-2503** [comment]
Uploading 2f66309f-ad95-42f0-9527-77645d457258.mp4…

**Comment by deepak05kr18** [comment]
> I have added the hover effects on user profile dashboard please review the PR.

@pankaj-2503 please refer to contribution guidelines before generating PR, as I have raised the issue and I am looking forward to work on it, mentors would first assign it to me and moreover according to given guidelines one can only generate the PR if it is assign to them by mentors. Hope you understand . You can close your PR and can work on those issues which would be assign to you or can raise a new issue.

**Comment by tanmoysrt** [comment]
Those 3 buttons looks good as it is. Because those are more likely as a tab switcher instead of a action button.

WDYT ?? @Arnabdaz @VaibhavUpreti @deepak05kr18

**Comment by deepak05kr18** [comment]
> Those 3 buttons looks good as it is. Because those are more likely as a tab switcher instead of a action button.
> 
> WDYT ?? @Arnabdaz @VaibhavUpreti @deepak05kr18

Yes, for buttons like tab switcher we don't need much hover but what if we put basic hover of just upward movement by few pixels? Then it would provide good visual quality.

**Comment by tanmoysrt** [comment]
> > Those 3 buttons looks good as it is. Because those are more likely as a tab switcher instead of a action button.
> > WDYT ?? @Arnabdaz @VaibhavUpreti @deepak05kr18
> 
> Yes, for buttons like tab switcher we don't need much hover but what if we put basic hover of just upward movement by few pixels? Then it would provide good visual quality.

We can change the background color on hover, maybe little darker green. On a tab bar, having upward movement is not good from user experience.

**Comment by deepak05kr18** [comment]
> > > Those 3 buttons looks good as it is. Because those are more likely as a tab switcher instead of a action button.
> > > WDYT ?? @Arnabdaz @VaibhavUpreti @deepak05kr18
> > 
> > 
> > Yes, for buttons like tab switcher we don't need much hover but what if we put basic hover of just upward movement by few pixels? Then it would provide good visual quality.
> 
> We can change the background color on hover, maybe little darker green. On a tab bar, having upward movement is not good from user experience.

Yes, sure we can do so

**Comment by deepak05kr18** [comment]
Hi @tanmoysrt , could you assign me this , or shall I proceed with a PR.

**Comment by tanmoysrt** [comment]
> Hi @tanmoysrt , could you assign me this , or shall I proceed with a PR.

You can update this issue

**Comment by Harishchandra-26** [comment]
I would like to work on this issue.
